tests/integration/api/test_rulebook.py

```diff
@@ -428,12 +491,46 @@ def test_list_actions_from_audit_rule_filter_name(client: APIClient, init_db):
     ]
 
 
+@pytest.mark.parametrize(
+    "ordering_field",
+    ["name", "status", "url", "fired_at", "rule_fired_at"],
+)
+@pytest.mark.django_db
+def test_list_actions_from_audit_rule_ordering(
+    client: APIClient, init_db, ordering_field
+):
+    audit_rule_id = init_db.audit_rule_1.id
+    response = client.get(
+        f"{api_url_v1}/audit-rules/{audit_rule_id}/actions/?ordering="
+        f"{ordering_field}"
+    )
+    assert response.status_code == status.HTTP_200_OK
+    actions = response.data["results"]
+    assert len(actions) == 2
+    assert (
+        actions[0][ordering_field]
+        == init_db.audit_action_1.__dict__[ordering_field]
```

Comment:
Should be using `getattr` rather than directly accessing `__dict_`.
Same comment for the other uses of `__dict__`.

Reply:
Updated.